products/clickhouse.md

```diff
@@ -0,0 +1,76 @@
+---
+title: ClickHouse
+addedAt: 2025-10-26
+category: database
+iconSlug: clickhouse
+permalink: /clickhouse
+versionCommand: clickhouse-client --version
+releasePolicyLink: https://clickhouse.com/docs/faq/operations/production#how-to-choose-between-clickhouse-releases
+changelogTemplate: https://github.com/ClickHouse/ClickHouse/releases/tag/v__LATEST__
```

Comment:
```suggestion
changelogTemplate: https://github.com/ClickHouse/ClickHouse/blob/master/CHANGELOG.md
```

Tags are suffixed with either `stable` or `lts`, so generated links will be wrong. Moreover changelog is available at https://github.com/ClickHouse/ClickHouse/blob/master/CHANGELOG.md.